src/style.css

```diff
@@ -1 +1,76 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
I see the padding and the margin are still here. Could you please remove it from the `*` selector and only apply it to the selectors that actually need them?